benchmark/collective/resnet/scripts/train_gpu.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-
-export FLAGS_sync_nccl_allreduce=1
+export CUDA_VISIBLE_DEVICES=4,5,6,7
```

Comment:
wired environment settings, why not 0,1,2,3

Reply:
These test lines will cleanup. Thanks!